src/PDFDoc.php

```diff
@@ -450,7 +450,7 @@ protected function _generate_signature_in_document() {
                 "Subtype" => "/Widget",
                 "FT" => "/Sig",
                 "V" => new PDFValueString(""),
-                "T" => new PDFValueString('Signature' . $this->_appearance['name'] ?? get_random_string()),
+                "T" => new PDFValueString(($this->_appearance['name'] ?? ('Signature' . get_random_string()))),
```

Comment:
So many parentheses are not necessary
```suggestion
                "T" => new PDFValueString($this->_appearance['name'] ?? ('Signature' . get_random_string())),
```